docs/tutorials/03_emulation_sensitivity.ipynb

```diff
@@ -0,0 +1,516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Scaling Sensitivity Analysis with Emulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Context\n",
```

Comment:
I'd reorder and slightly reword the intro section. Specifically, I'd remove context (people who are reading the autoemulate docs already know this) and start with purpose of the tutorial instead i.e., explain why someone reading the docs should read this page. Some of the context text explain gin the cardio side might need to be moved elsewhere.

I made some rough edit suggestions to this end but they are not necessarily comprehensive.